server/src/main/java/com/google/fhir/gateway/BearerAuthorizationInterceptor.java

```diff
@@ -211,7 +211,7 @@ private boolean sendGzippedResponse(ServletRequestDetails requestDetails) {
     if (acceptEncodingValue == null) {
       return false;
     }
-    return GZIP_ENCODING_VALUE.equalsIgnoreCase(acceptEncodingValue);
+    return acceptEncodingValue.contains(GZIP_ENCODING_VALUE);
```

Comment:
As you suggested, I think it is better to do case-insensitive comparison. Please also update/add unit-tests [here](https://github.com/google/fhir-gateway/blob/4376a58fa2e1316050177c3bb3d206f1f1fcd7c1/server/src/test/java/com/google/fhir/gateway/BearerAuthorizationInterceptorTest.java#L311).